src/plugins/intel_gpu/src/kernel_selector/kernels/sdpa/sdpa_kernel_micro.cpp

```diff
@@ -205,7 +206,8 @@ void SDPAKernelMicro::init_microkernels(const sdpa_params& params, micro::Packag
             break;
         }
         case gpu_arch::xe_hpc:
-        case gpu_arch::xe2: {
+        case gpu_arch::xe2:
+        case gpu_arch::xe3: {
```

Comment:
Please add a check in Validate() method that arch it not `unknown`.

Reply:
Check 'unknown' is already in Validate().
`
if (params.engineInfo.arch < gpu_arch::xe_hpg || !params.engineInfo.supports_microkernels)
        return false;
`

Reply:
Hmm, then why did micro sdpa crash for xe3?